### DISCUSSION TITLE
How do i run a project and get the exit code of the project that i ran.

### DISCUSSION THREAD

**Discussion by ZNackasha** [nrwl/nx#22982] [General]
I am using nx for a mono repo. One of the apps I have in nx is run during the CI. Before version 18 of nx. when you ran/serve a js node app the exit code would be reflected in the terminal after the command.  now it seems like the exit code is always 0. 

here is the command that I use

nx serve myApp --inspect=false --prod --args="some-args-here" --watch=false

is there a way to get the underline exit code.

**Comment by jaysoo** [comment]
We'll need to update the executor to respect the exit code of the application.
